src/meta/client/src/lib.rs

```diff
@@ -37,7 +37,10 @@ pub static METACLI_COMMIT_SEMVER: Lazy<Version> = Lazy::new(|| {
 
     let semver = semver.strip_prefix('v').unwrap_or(semver);
 
-    Version::parse(semver).unwrap_or_else(|e| panic!("Invalid semver: {:?}: {}", semver, e))
+    Version::parse(semver).expect(
+        "run `git fetch --tags https://github.com/datafuselabs/databend` to slove this error, 
```

Comment:
Not need the repo name https://github.com/datafuselabs/databend?

Reply:
IMHO, new contributors may create their own fork and have not set this repository as upstream yet, so fetch with repo name is a general method to fetch newest tags.

Reply:
When the user meet the error, it means that he has clone the repo but not fetch the tags, now only run `git fetch --tags` is ok.